### ISSUE TITLE
Changelogs

### ISSUE THREAD

**Issue by dbrgn** [chinedufn/percy#61]
Hi @chinedufn. It would be great to have a changelog per crate. Reasoning and an example format can be found at https://keepachangelog.com/. Here's a simplified version of that: https://github.com/dbrgn/tealdeer/blob/master/CHANGELOG.md

The CHANGELOG is a great place to let users of the crate know about changes between versions. This is especially handy for upgrade notes.

**Comment by chinedufn** [comment]
@dbrgn great idea!

So one thing here is that Percy is still in the experimental phase .. but we can just make that explicitly clear at the top of the CHANGELOG

Just so that people don't get the impression that this is a stable project or anything.

Cool - great idea - will add!

**Comment by dbrgn** [comment]
Sure, Semantic Versioning explicitly states that below 1.0.0 anything goes :slightly_smiling_face:

**Comment by chinedufn** [comment]
Yeah for sure! I’ve just noticed other pre 1.0 libraries paying a lot of attention to stability so we’ll want to just call that out.

👍🏿

**Comment by chinedufn** [comment]
Added a CHANGELOG to `virtual_dom_rs`

https://github.com/chinedufn/percy/commit/b2adfdc47f78f33c1cb94f0b3648f212769ccbff

**Comment by chinedufn** [comment]
Added a CHANGELOG for `router-rs`

a52fd99
